CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
 - safe way to help with rollbacks?
 - run `kubectl describe pod`
 - `kubectl` apply a buffer to current context/namespace
+- ansi-term support for `kubectl exec`
```

Comment:
it's confusing and I need to change how the changelog is formatted, but this should go under the 3.0 section. The idea is that if I were to release and tag 3.0, it would be a part of it. "Unreleased" is a draft section for maybe upcoming features.

Reply:
Oops! I updated